v2/cli/init_command.go

```diff
@@ -0,0 +1,158 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strings"
+	"text/template"
+
+	"github.com/AlecAivazis/survey/v2"
+	"github.com/brigadecore/brigade/v2/internal/file"
+	"github.com/urfave/cli/v2"
+)
+
+type InitFields struct {
+	ID       string
+	Language string
+	Git      string
+}
+
+var initCommand = &cli.Command{
+	Name:  "init",
+	Usage: "Initialize a template Brigade project",
+	Flags: []cli.Flag{
+		&cli.StringFlag{
+			Name:    flagID,
+			Aliases: []string{"i"},
+			Usage: "Name of the Brigade project to create " +
+				"(required)",
+			Required: true,
+		},
+		&cli.StringFlag{
+			Name:    flagLanguage,
+			Aliases: []string{"l"},
+			Usage: "Set the language of the brigade project file" +
+				"to either JavaScript or TypeScript (default)",
+		},
+		&cli.StringFlag{
+			Name:    flagGit,
+			Aliases: []string{"g"},
+			Usage:   "Initialize the brigade project from an existing git repo ",
+		},
+	},
+	Action: initialize,
+}
+
+// Handle init CLI flags
+func initialize(c *cli.Context) error {
+	var err error
+
+	projectID := c.String(flagID)
+	language := strings.ToLower(c.String(flagLanguage))
+	gitRepo := c.String(flagGit)
+
+	// Check if inputted language is valid
+	switch language {
+	case "ts", "typescript", "":
+		language = "ts"
+	case "js", "javascript":
+		language = "js"
+	default:
+		return fmt.Errorf("unrecognized value %q for --language flag", language)
```

Comment:
Would it be helpful to print the expected values for the language flag?

Reply:
`return fmt.Errorf("unrecognized value %q for --language flag (ts or js expected)", language)`

Good suggestion, how does this look?